docs/getting_started/install_primehub_ce.md

```diff
@@ -14,105 +14,123 @@ sidebar_label: Install PrimeHub CE
 
 The document assumes there is a PrimeHub-ready-Kubernetes already. The steps below are performed against the cluster.
 
-## Prepare the Value File
+## Prepare the environment variables
 
-Prepare the value file `primehub-values.yaml` for helm installation.
+Prepare the variables before installation
 
-> Pure IP address is not allowed because we use Kubernetes ingress to route the traffic. A trick is to use the [nip.io](https://nip.io/) service to map an IP to a domain name. (e.g., `1.2.3.4.nip.io`)
+Key | Description | Example value
+----|-------------| ----
+`PRIMEHUB_DOMAIN` | The domain name of PrimeHub which user can access to. | `example.primehub.io`
+`PH_PASSWORD` | PrimeHub admin `phadmin` default password| `yourDefaultPassw0rd`
+`KC_PASSWORD` | Keycloak admin `keycloak` default password | `yourDefaultPassw0rd`
 
-Key | Description
-----|------------------------------------
-`PRIMEHUB_DOMAIN` | The domain name of PrimeHub. It can be the same as PrimeHub's one.
 
+## Clone the PrimeHub install repository
 
+```bash
+git clone https://github.com/InfuseAI/primehub.git
+```
 
-Modify the environment variables below and execute the commands to generate the value file.
+## Validate the primehub-install command is executable
 
-```
-PRIMEHUB_DOMAIN=1.2.3.4.nip.io
-
-cat <<EOF > primehub-values.yaml
-primehub:
-  domain: ${PRIMEHUB_DOMAIN}
-ingress:
-  annotations:
-    kubernetes.io/ingress.allow-http: "true"
-    nginx.ingress.kubernetes.io/ssl-redirect: "false"
-  hosts:
-  -  ${PRIMEHUB_DOMAIN}
-EOF
+```bash
+./primehub/install/primehub-install
 ```
 
-## Add the chart repository
-
+You should see the usage message like this:
 ```bash
-helm repo add infuseai https://charts.infuseai.io
+USAGE:
+  primehub-install create  singlenode  [options]            : Create single-node k8s environment
+  primehub-install status  singlenode                       : Show the statuse of single-node k8s environment
+  primehub-install destroy singlenode                       : Destroy single-node k8s environment
+...
 ```
 
+## Install PrimeHub required binaries
+
 ```bash
-helm repo update
+./primehub/install/primehub-install required-bin
 ```
 
-## Install
+This will install the required commands onto `~/bin`. You should append the `~/bin` to your `PATH` variables.
 
-1. Run Helm command to install PrimeHub
+## Install PrimeHub 
 
-   ```
-   helm upgrade \
-     primehub infuseai/primehub \
-     --install \
-     --create-namespace \
-     --namespace hub  \
-     --timeout 30m  \
-     --values primehub-values.yaml
+Prepare two terminals, one to execute the primehub install script, the other to monitor the install progress by watching the pods status.
+
+### Terminal one
+
+Install by `primehub-install create primehub` and specify the version. ex. `v3.6.2`.
```

Comment:
add one sentence 
"Please check the latest stable version."

Reply:
add